lib/SPIRV/LLVMToSPIRVDbgTran.cpp

```diff
@@ -1219,6 +1243,28 @@ SPIRVEntry *LLVMToSPIRVDbgTran::transDbgFuncDefinition(SPIRVValue *FuncDef,
                         Ops, BB, BB->getInst(0));
 }
 
+SPIRVEntry *LLVMToSPIRVDbgTran::transDbgEntryPoint(const DISubprogram *Func,
+                                                   SPIRVEntry *DbgFunc) {
+  if (!isNonSemanticDebugInfo())
+    return nullptr;
+
+  SPIRVWordVec Ops(SPIRVDebug::Operand::EntryPoint::OperandCount);
+
+  DICompileUnit *CU = Func->getUnit();
+  StringRef Producer = CU->getProducer();
+  StringRef Flags = CU->getFlags();
+
+  auto It = MDMap.find(CU);
+  SPIRVEntry *CUVal = It != MDMap.end() ? It->second : getDebugInfoNone();
+
+  using namespace SPIRVDebug::Operand::EntryPoint;
```

Comment:
nit:
can be moved before
`SPIRVWordVec Ops(SPIRVDebug::Operand::EntryPoint::OperandCount);`